front-end/src/app/components/user-profile/user-profile.component.scss

```diff
@@ -59,10 +59,9 @@
     display: inline-block;
     padding: 20px;
     font-size: 36px;
-    position: relative;
-    bottom: 10px;
   }
   img {
     border-radius: 15%;
+    vertical-align: -10px;
```

Comment:
you don't use pixel values for vertical-align - see https://css-tricks.com/almanac/properties/v/vertical-align/ for examples on how to align inlign images with text

Reply:
Ahhh, thank you. I get it now. I also fixed my alignment issues. For some strange reason, there was a margin on the h1 tag screwin' things up. Not fun.